src/errors/VerificationError.coffee

```diff
@@ -1,6 +1,8 @@
 class VerificationError extends Error
   constructor: (@code, @description) ->
     super()
+    code = ""
```

Comment:
They are:
https://github.com/sjt88/passport-eveonline/blob/84246fb186a5ee575670a7e44cf3bb938beadec3/src/strategy.coffee#L43

However i'm unsure why I've set their values here to an empty string - it looks unnecessary to me. I'll check if it's necessary